pkg/bootstrapper/bootstrapper.go

```diff
@@ -350,6 +350,17 @@ func (wmcb *winNodeBootstrapper) initializeKubeletFiles() error {
 			dest: filepath.Join(wmcb.installDir, "kubelet-ca.crt"),
 		},
 	}
+
+	// Create the manifest directory needed by kubelet for the static pods, we shouldn't override if the pod manifest
+	// directory already exists
+	podManifestDirectory := filepath.Join(wmcb.installDir, "etc\\kubernetes\\manifests")
```

Comment:
This should probably be changed to `filepath.Join(wmcb.installDir, "etc", "kubernetes", "manifests")` since we're trying to remove hardcoded path seperators. https://github.com/openshift/windows-machine-config-operator/pull/102#discussion_r356205495

Reply:
Done